consensus/hotstuff/eventhandler/event_handler.go

```diff
@@ -196,19 +196,19 @@ func (e *EventHandler) OnReceiveProposal(proposal *model.Proposal) error {
 
 // TimeoutChannel returns the channel for subscribing the waiting timeout on receiving
 // block or votes for the current view.
-func (e *EventHandler) TimeoutChannel() <-chan time.Time {
+func (e *EventHandler) TimeoutChannel() <-chan model.TimerInfo {
 	return e.paceMaker.TimeoutChannel()
 }
 
 // OnLocalTimeout handles a local timeout event by creating a model.TimeoutObject and broadcasting it.
 // No errors are expected during normal operation.
-func (e *EventHandler) OnLocalTimeout() error {
+func (e *EventHandler) OnLocalTimeout(timerInfo model.TimerInfo) error {
 	curView := e.paceMaker.CurView()
 	e.log.Debug().Uint64("cur_view", curView).Msg("timeout received from event loop")
 	// TODO(active-pacemaker): update telemetry
 	defer e.notifier.OnEventProcessed()
 
-	err := e.broadcastTimeoutObjectIfAuthorized()
+	err := e.broadcastTimeoutObjectIfAuthorized(timerInfo.Tick)
```

Comment:
if we followed [my previous comment](https://github.com/onflow/flow-go/pull/3430#discussion_r1016067102), I don't think we would need the `timerInfo.Tick` here anymore.